de.metas.swat/de.metas.swat.base/src/main/java/de/metas/invoicecandidate/api/IInvoiceCandDAO.java

```diff
@@ -76,7 +76,12 @@
 	 */
 	<T extends I_C_Invoice_Candidate> Iterator<T> retrieveInvoiceCandidates(IQueryBuilder<T> queryBuilder);
 
-	/** Return a list of ICs for each query included in the given {@code multiQuery}. */
+	List<I_C_Invoice_Candidate> getByQuery(InvoiceCandidateMultiQuery multiQuery);
+
+	/**
+	 * @deprecated Please use {@link #getByQuery(InvoiceCandidateMultiQuery)} or introduce another method is internally is calling this one
+	 */
+	@Deprecated
```

Comment:
here i agree with @teosarca ; pls make the method private and remove it from `IInvoiceCandDAO`

Reply:
refactored, deleted method form interface and made it private in impl.

Reply:
done